tree_test.go

```diff
@@ -154,6 +154,12 @@ func TestTreeWildcard(t *testing.T) {
 		"/info/:user/public",
 		"/info/:user/project/:project",
 		"/info/:user/project/golang",
+		"/aa/*xx",
+		"/ab/*xx",
+		"/:cc",
+		"/:cc/cc",
+		"/get/test/abc/",
+		"/get/:param/abc/",
```

Comment:
can you add more routes with multiple params?
I also want to confirm something like this:
```
/something/:paramname/thirdthing
/something/secondthing/test
```

If you someone does `GET /something/secondthing/thirdthing` that should have a 404

Reply:
Thank you for your review. I need some time to do the test. The final result should be 404 is correct

Reply:
hi @rw-access 
1. Added more unit tests
2. This scene I tested ` GET/something/secondthing thirdthing ` matching ` / something / : paramname/thirdthing ` is correct
    When n.indices matches a child node, 'line 271 n.hildren [len(n.hildren)-1]' will be executed if it does not
```
`GET /something/secondthing/abc` 404
`GET /something/secondthing/ff` 404
```
![n](https://user-images.githubusercontent.com/67568757/124377236-96c04c00-dcdd-11eb-931b-fc3b6f9b4fc7.png)